content/efcore/getting-started/README.md

```diff
@@ -12,4 +12,5 @@ This tutorial is aimed at people who are learning EF Core for the first time. Mo
 ## Topics Covered
 
 This tutorial will cover the following topics:
-- [Intro to EF Core](what-is-ef-core.md)
\ No newline at end of file
+- [Intro to EF Core](what-is-ef-core.md)
```

Comment:
I should have caught this in PR #20, but the url should be `intro-to-ef-core.md`